test/units/modules/remote_management/oneview/test_oneview_datacenter_facts.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) 2016-2017 Hewlett Packard Enterprise Development LP
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from ansible.compat.tests import unittest
```

Comment:
Please update tests to not use unittest, and instead use pytest style tests.

Reply:
Is this a requirement listed somewhere? Most of the tests for our modules are already implemented using unittest

Reply:
@fgbulsoni http://docs.ansible.com/ansible/devel/dev_guide/testing_units.html
> Ansible drives unit tests through pytest. This means that tests can either be written a simple functions which are included in any file name like test_<something>.py or as classes.